src/app/backend/resource/networkpolicy/detail.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2017 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package networkpolicy
```

Comment:
Is it copy from Kubernetes repo?

Reply:
yeah ,i think it is an efficient and convenient way for a beginner

Reply:
But we cannot merge it if it is a pure copy. We need to use custom structures and methods to extract only that information that we need.

Reply:
sorry,There are some misunderstandings.My angularJs not very well at first.so i refer to kubernets ingress and pod module.but not cpoy the module.i finished the module step by step.and the license is copy from other module not from license/header.txt .

Reply:
very thanks. i do not understand the propose of the dashboard project." We would like to display all data that kubectl get/describe does and that is why we create our structures."i think dashboard project provides more information and operation to a user than kubectl command .The design of the project is used to replace the kubectl command.The dashboard project should provide more convenient tools to a user not only a copy of kubctl.so i integrated many functions like helm ,Prometheus ...

Reply:
ok ,very thanks.

Reply:
@zhangjianweibj Yes, it should, but first let's start with the basics, not the everything. Can you prove that you are displaying all these data in the UI?